Docs

rocketraman opened the pull request.

jvz commented:
Looking good so far! The only other important generated report to include in the site somehow would be the result of the apache-rat report. That generates a txt file, so I wonder if we can turn that into a source for AsciiDoc, then just `include:rat.txt[]` them or similar.

rocketraman commented:
> The only other important generated report to include in the site somehow would be the result of the apache-rat report. That generates a txt file, so I wonder if we can turn that into a source for AsciiDoc, then just include:rat.txt[] them or similar.

@jvz Done! I've added the rat check in pull #3, merged that branch here, and added the inclusion of it into the generated docs on this branch. This branch will merge cleanly if pull #3 is merged first (as it should be).

jvz commented:
I added a travis build config if you want to rebase from master. Should fix this bot error.
